lighthouse-viewer/app/src/psi-api.js

```diff
@@ -0,0 +1,50 @@
+/**
+ * @license Copyright 2019 Google Inc. All Rights Reserved.
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with the License. You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the specific language governing permissions and limitations under the License.
+ */
+'use strict';
+
+/** @typedef {{lighthouseResult?: LH.Result, error?: {message: string}}} PSIResponse */
+
+const PSI_URL = 'https://www.googleapis.com/pagespeedonline/v5/runPagespeed';
+const PSI_KEY = 'AIzaSyAjcDRNN9CX9dCazhqI4lGR7yyQbkd_oYE';
+const PSI_DEFAULT_CATEGORIES = [
+  'performance',
+  'accessibility',
+  'seo',
+  'best-practices',
+  'pwa',
+];
```

Comment:
this also needs to be kept in sync with master

Reply:
done, added a test for this. couldn't just use `Config` directly b/c:

1) the build system is weird here, `require` didn't work and I didn't look further than that. Should consider updating this one day so we can use async/await.

2) the order of these categories matters for the PSI api, so best to just declare the array manually.